src/modules/feature-modules/admin/pages/users/user-info.component.ts

```diff
@@ -69,94 +52,35 @@ export class PageUserInfoComponent extends CoreComponent implements OnInit {
-          !response.lockedAt
-        ) {
-          this.titleActions.push({
-            type: 'link',
-            label: 'Change role',
-            url: `/admin/users/${this.user.id}/service-users/change-role`
-          });
-
-          this.unitLength = organisations.filter(org => (response.userOrganisations[0].id === org.id))[0].organisationUnits.length;
-        }
-
-        if (response.type === UserRoleEnum.ACCESSOR || response.type === UserRoleEnum.QUALIFYING_ACCESSOR) {
-          if (response.userOrganisations.length > 0) {
-            this.sections.userInfo = [
-              { label: 'Name', value: response.displayName },
-              { label: 'Type', value: 'Authorised person' },
-              { label: 'Role', value: this.stores.authentication.getRoleDescription(response.userOrganisations[0].role) },
-              { label: 'Email address', value: response.email },
-              { label: 'Phone number', value: response.phone ? response.phone : ' NA' },
-              { label: 'Account status', value: !response.lockedAt ? 'Active' : 'Locked' }
-            ];
-          }
-          this.sections.organisation = response.userOrganisations;
-        }
-        else if ([UserRoleEnum.ASSESSMENT, UserRoleEnum.INNOVATOR].includes(response.type)) {
-          this.sections.userInfo = [
-            { label: 'Name', value: response.displayName },
-            { label: 'Type', value: this.stores.authentication.getRoleDescription(response.type) },
-            { label: 'Email address', value: response.email },
-            { label: 'Phone number', value: response.phone ? response.phone : ' NA' },
-            { label: 'Account status', value: !response.lockedAt ? 'Active' : 'Locked' }
-          ];
-          if (response.type === UserRoleEnum.INNOVATOR) {
-            this.sections.innovations = response.innovations.map(x => x.name);
-            if (response.userOrganisations.length > 0 && !response.userOrganisations[0].isShadow) {
-              this.sections.userInfo = [...this.sections.userInfo,
-              { label: 'Company name', value: response.userOrganisations[0].name },
-              { label: 'Company size', value: response.userOrganisations[0].size }
-              ];
+    this.usersService.getUserInfo(this.activatedRoute.snapshot.params.userId).subscribe({
+      next: (response) => {
+        this.user = {
+          ...response,
+          rolesDescription: response.roles.map(r => {
+            let roleDescription = this.stores.authentication.getRoleDescription(r.role);
+            if (r.role === UserRoleEnum.ASSESSMENT && r.displayTeam) {
+              roleDescription = `${roleDescription} (${r.displayTeam})`;
+            }
+            if (r.role === UserRoleEnum.QUALIFYING_ACCESSOR || r.role === UserRoleEnum.ACCESSOR) {
+              roleDescription = `${roleDescription} (${r.organisationUnit?.name})`;
+            }
```

Comment:
Here you can make just one validation since when the user is QA/A the `displayTeam` will return the unit name:

```
let roleDescription = this.stores.authentication.getRoleDescription(r.role);
if(r.displayTeam) {
    roleDescription += ` (${r.displayTeam})`;
}
```

Reply:
I didn't notice this. Thanks Diogo!